cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -67,16 +72,23 @@ public static void main(String[] args) {
         Vertx vertx = Vertx.vertx();
         KubernetesClient client = new DefaultKubernetesClient();
 
-        isOnOpenShift(vertx, client).setHandler(os -> {
-            if (os.succeeded()) {
-                run(vertx, client, os.result().booleanValue(), config).setHandler(ar -> {
-                    if (ar.failed()) {
-                        log.error("Unable to start operator for 1 or more namespace", ar.cause());
+        createClusterRoles(vertx, config, client).setHandler(crs -> {
```

Comment:
just a nit ... we are using methods starting with `maybe` when the action may not happen, so maybe :-D we could have `maybeCreateClusterRoles`. Btw no need to change, it's fine with this name as well.